packages/x11-repo/build.sh

```diff
@@ -2,25 +2,33 @@ TERMUX_PKG_HOMEPAGE=https://github.com/termux/x11-packages
 TERMUX_PKG_DESCRIPTION="Package repository containing X11 programs and libraries"
 TERMUX_PKG_LICENSE="Apache-2.0"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION=8.4
-TERMUX_PKG_REVISION=1
+TERMUX_PKG_VERSION=8.5
 TERMUX_PKG_AUTO_UPDATE=false
 TERMUX_PKG_DEPENDS="termux-keyring"
 TERMUX_PKG_SKIP_SRC_EXTRACT=true
 TERMUX_PKG_PLATFORM_INDEPENDENT=true
```

Comment:
```suggestion
TERMUX_PKG_PLATFORM_INDEPENDENT=true
TERMUX_PKG_CONFFILES="
etc/apt/sources.list.d/x11.sources
"

Reply:
I guess that's something we could do.

Reply:
Actually, no it's not something we should do.
`CONFFILES` aren't removed by a regular uninstall of a package.
Meaning `pkg rm tur-repo` would leave the sources file in place.
In effect not actually removing the repo.

Reply:
Sorry, my bad.